docs/html/chip/chip.html

```diff
@@ -0,0 +1,3 @@
+<div class="ray-chip">
```

Comment:
can you please add the tab index attribute to the html examples?

Reply:
yup